src/solver/simulation/sim_structure_probleme_economique.h

```diff
@@ -505,8 +505,6 @@ struct AdequacyPatchParameters
     float ThresholdDisplayLocalMatchingRuleViolations;
```

Comment:
Use double for `ThresholdInitiateCurtailmentSharingRule` and `ThresholdDisplayLocalMatchingRuleViolations`